### PR TITLE
Aave V3 OP exit isolation mode - Review

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: src/20230924_AaveV3_Opt_OPRiskParametersUpdate/AaveV3_Optimism_OPRiskParametersUpdate_20230924.t.sol [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    );
+
+    GovHelpers.executePayload(vm, address(proposal), AaveGovernanceV2.OPTIMISM_BRIDGE_EXECUTOR);
+
+    ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot(
+      'postAaveV3_Optimism_OPRiskParametersUpdate_20230924',
+      AaveV3Optimism.POOL
+    );
+
+    _noReservesConfigsChangesApartFrom(
```
Comment (Zer0dot) [inline src/20230924_AaveV3_Opt_OPRiskParametersUpdate/AaveV3_Optimism_OPRiskParametersUpdate_20230924.t.sol:36]: Looks like the tests are not testing that the configuration is as expected, only that the e2e test works and no other configs were changed. Would be good to include that.

Comment (marczeller) [reply src/20230924_AaveV3_Opt_OPRiskParametersUpdate/AaveV3_Optimism_OPRiskParametersUpdate_20230924.t.sol:41]: tests have been reworked and improved.

File: src/20230924_AaveV3_Opt_OPRiskParametersUpdate/AaveV3_Optimism_OPRiskParametersUpdate_20230924.sol [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * @author Marc Zeller - Aave-Chan Initiative
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x617adb838ce95e319f06f72e177ad62cd743c2fe3fd50d6340dfc8606fbdd0b3
+ * - Discussion: https://governance.aave.com/t/arfc-op-risk-parameters-update-aave-v3-optimism-pool/14633
+ */
+contract AaveV3_Optimism_OPRiskParametersUpdate_20230924 is AaveV3PayloadOptimism {
+  uint256 public constant BORROW_CAP = 500_000;
+  address public constant INTEREST_RATE_STRATEGY = 0x3B57B081dA6Af5e2759A57bD3211932Cb6176997;
+  uint256 public constant DEBT_CEILING = 0;
+
+  function _preExecute() internal override {
```
Comment (kyzia551) [inline src/20230924_AaveV3_Opt_OPRiskParametersUpdate/AaveV3_Optimism_OPRiskParametersUpdate_20230924.sol:18]: Please change IR strategy update to use ConfigEngine, similar to [this example](https://github.com/bgd-labs/aave-proposals/blob/622f2741d687033ed890d51506bd8c0222fcf883/src/AaveV2PolygonRatesUpdates_20230614/AaveV2PolygonRatesUpdates_20230614.sol#L46)

**Review by kyzia551 — CHANGES_REQUESTED** [review]

File: src/20230924_AaveV3_Opt_OPRiskParametersUpdate/AaveV3_Optimism_OPRiskParametersUpdate_20230924.sol (hunk 2 above)
Comment (marczeller) [reply src/20230924_AaveV3_Opt_OPRiskParametersUpdate/AaveV3_Optimism_OPRiskParametersUpdate_20230924.sol]: i've rm rateStrategy update, it's not needed

**Review by kyzia551 — DISMISSED** [review]

**Review by Zer0dot — APPROVED** [review]

**Review by kyzia551 — APPROVED** [review]

**Pull request merged**